torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -880,7 +880,11 @@ def clone_args(self, *args, **kwargs) -> Tuple[List[Any], Dict[str, Any]]:
 
     def benchmark_all_configs(self, *args, **kwargs):
         with dynamo_timed(
-            "CachingAutotuner.benchmark_all_configs", log_pt2_compile_event=True
+            "CachingAutotuner.benchmark_all_configs",
+            log_pt2_compile_event=True,
+            metadata={"kernel_name": self.inductor_meta.get("kernel_name", None)},
+            dynamo_compile_column_us="runtime_triton_autotune_time_us",
+            compile_id=self.inductor_meta.get("compile_id", None),
```

Comment:
The compile_id here is the other half of a comment I made above. I save away the compile_id in the inductor_meta as part of the codegen, then retrieve it here and pass to dynamo_timed.